### ISSUE TITLE
Sequence / Interaction flow: When are resources created and when is access given?

### ISSUE THREAD

**Issue by uvdsl** [mandat-project/hackathon-demo#9]
I am wondering though if we should rethink certain aspects of the current situation, e.g.,
Max is given access to the data request resource BEFORE there even exists a data request by the Bank.
For the demo, this may be sufficient, however, wouldn't it be more accurate to only give access to Max (or any other) for that resource, only after this resource actually becomes relevant for Max?
(This was shortly discussed at the end of the hackathon, and left for potential future work)
